pyproject.toml

```
@@ -60,3 +61,24 @@ requires = [
     "versioneer-518"
 ]
 build-backend = "setuptools.build_meta"
+
+[tool.pytest.ini.options]
```

```suggestion
[tool.pytest.ini_options]
```
https://docs.pytest.org/en/stable/reference/customize.html#pyproject-toml